src/components/KonnectorInstall.jsx

```diff
@@ -38,7 +39,8 @@ export const KonnectorInstall = ({
   isFetching,
   isValid,
   isSuccess,
-  dirty
+  dirty,
+  successButtonLabel
```

Comment:
Wow, that's a LOT of props 😮

Reply:
This part of collect is a real nightmare, we aim to rewrite this part as soon as possible.